app/src/ereferrals/java/org/eyeseetea/malariacare/data/sync/importer/PullOrganisationCredentialsController.java

```diff
@@ -0,0 +1,74 @@
+package org.eyeseetea.malariacare.data.sync.importer;
+
+import android.content.Context;
+
+import org.eyeseetea.malariacare.data.IDataSourceCallback;
+import org.eyeseetea.malariacare.data.database.model.Program;
+import org.eyeseetea.malariacare.data.database.utils.PreferencesEReferral;
+import org.eyeseetea.malariacare.data.remote.PullOrganisationCredentials;
+import org.eyeseetea.malariacare.domain.entity.Credentials;
+
+public class PullOrganisationCredentialsController {
```

Comment:
This class is really a OrgUnitRepository and should return domain OrgUnits retrieving from API DHIS using SeverAPIController. Can you Rename and create a interface in domain for apply dependency inversion from UseCase where is invoked?